Added `load` output event property to `markdown.component`

coolwr opened the pull request:
Implemented `load` output event property on `markdown.component` to emit when source is loaded.

jfcere commented:
Hi @coolwr, I am trying to understand some things about your PR, hopefully you'll be able to lighten me up.

1. From what I see the `load` event will be emit everytime the markdown is rendered (using variable with `data` input property, when using transclusion, and fetching remove data). If the purpose is to emit an event when the `src` input binding is used to fetch remote files why not emit the event in the subscribe section of the `src` input binding?

2. Why emit `nativeElement` ? I trying to understand the context in which you would need this... also according to what you wrote the `EventEmitter` typing is wrong as `nativeElement` is in fact an `HTMLElement` so the emitter should be typed `EventEmitter<HTMLElement>`.

coolwr commented:
Hi @jfcere , thanks for feedback! I've updated the PR so that the `load` event emits only for the `src` input binding is done fetching remote files. 

Also, instead of emitting the HTMLElement, I changed it to emit the loaded markdown string. It is not necessary for my implementation, but could be useful for those looking to get the markdown value.

jfcere commented:
Great, this looks good 👍

jfcere commented:
As a follow up, [v6.1.0](https://github.com/jfcere/ngx-markdown/releases/tag/v6.1.0) has been released which includes your PR.
